src/redux/store.test.ts

```diff
@@ -25,6 +27,16 @@ const resetStateOnInvalidStoredAccount = jest.spyOn(
 
 const loggerErrorSpy = jest.spyOn(Logger, 'error')
 
+const getNonApiReducers = <R = Omit<RootState, ApiReducersKeys>>(state: RootState): R => {
+  const apiReducersKeys: string[] = ['transactionFeedV2Api'] satisfies ApiReducersKeys[]
+  return Object.entries(state).reduce((acc, [reducerKey, value]) => {
+    const key = reducerKey as keyof R
+    if (apiReducersKeys.includes(reducerKey)) return acc
+    acc[key] = value as unknown as any
+    return acc
+  }, {} as R)
```

Comment:
nit: probably ok here because it's short, but I try to avoid `reduce` as it can often lead to hard to read code.

Reply:
@jeanregisser there are also a few `.reduce` usages in the follow-up PRs as well. Though the reason is not as applicable in this particular case, but I'm trying to reduce the amount of iterations the paginated data gets processed over. Considering, the Transaction Feed screen is one of the main screens with potentially lots of data I thought it made sense to try to minimize iterations so it is as optimized as possible. 

I might be overthinking this so please let me know if that's the case. I've tried to make all the `.reduce` usages (the ones you'll see later in other PRs) fully descriptive but please let me know if there's anything that can be improved in that regard!

Reply:
Ok, I think we should optimize for readability first, and avoid `reduce` in general. Given the number of elements we're dealing with, iterating once, twice or more times usually won't have much real world impact.
And we can usually refactor to iterate once with a for loop, if performance is measured to be a problem.

Reply:
@jeanregisser Got it, I'll tweak code in the other PRs to follow this approach!

Reply:
@jeanregisser refactored to for loop